pom.xml

```diff
@@ -29,6 +29,16 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-websocket</artifactId>
+			<exclusions>
```

Comment:
why you removed `tomcat` and added `undertow`?

Reply:
@amarlearning Sorry, i forgot this in PR ... in my debug i tried to change to Undertow, to solve your issue, but the problem was continued ... this was indifference ... the fix of this issue is in Javascript

I recommend still use Undertow because he was thought to websocket

Reply:
> In my debug I tried to change to Undertow, to solve your issue, but the problem was continued.

@RenanPalmeira since this is not in the scope of the issue, let remove this change.

Reply:
@amarlearning Hey! i updated pom...